kwik-core/stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```

Comment:
ditto above, only need the `resolver` and `packages` directives.. the rest is noise

Reply:
actually, since this is core, you only need `resolver`